### PR TITLE
Runs loader

### PR DESCRIPTION
**Pull request by mostafa-mahmoud**

### CODE REVIEW

File: runnables.py [hunk 1]
```diff
@@ -137,6 +138,17 @@ def run_recommender(self):
...
 
+    def run_all_runs(self):
+        runs = RunsLoader()
+        for config_dict in runs.get_runnable_recommenders():
+            recommender = RecommenderSystem(abstracts_preprocessor=self.abstracts_preprocessor, ratings=self.ratings,
+                                            config=config_dict, verbose=self.verbose, load_matrices=self.load_matrices,
+                                            dump_matrices=self.dump, train_more=self.train_more)
+            recommender.train()
+            print(recommender.content_based, recommender.collaborative_filtering)
+            print(numpy.round(recommender.get_predictions(), 3))
```
Comment (ndomar) [inline runnables.py]: Not needed

File: runnables.py [hunk 2]
```diff
@@ -137,6 +138,17 @@ def run_recommender(self):
...
+            print("")
```
Comment (ndomar) [inline runnables.py]: why?

File: runnables.py [hunk 3]
```diff
@@ -137,6 +138,17 @@ def run_recommender(self):
         recommender.train()
         print(recommender.content_based.get_document_topic_distribution().shape)
 
+    def run_all_runs(self):
```
Comment (ndomar) [inline runnables.py]: Print which run are you running if verbose

File: runnables.py (hunk 1 above)
Comment (mostafa-mahmoud) [inline runnables.py]: Just showing it's running, will change the prints after #67

File: runnables.py (hunk 3 above)
Comment (mostafa-mahmoud) [inline runnables.py]: I'm printing which recommenders in line 148, will change the prints after #67

File: runnables.py (hunk 2 above)
Comment (mostafa-mahmoud) [inline runnables.py]: Just a separation between each run, will change the prints after #67

**Pull request merged**